app/models/manageiq/providers/ansible_tower/inventory/persister/automation_manager.rb

```diff
@@ -1,19 +1,18 @@
 class ManageIQ::Providers::AnsibleTower::Inventory::Persister::AutomationManager < ManagerRefresh::Inventory::Persister
+  include ManagerRefresh::Inventory::AutomationManager
+
+  has_inventory automation_manager.credentials
```

Comment:
I don't think you want to actually want to call `automation_manager.credentials` (or `cloud_manager.vms`) at class instantiation here.

Think the `automation_manager` is determined at runtime - no?

this is probably a runtime call not a class instantiation call